core/src/main/resources/jenkins/install/SetupWizard/setupWizardConfigureInstance.jelly

```diff
@@ -26,15 +26,15 @@ THE SOFTWARE.
 
   <l:layout norefresh="true" type="full-screen" title="${it.displayName}">
     <l:main-panel>
-     <div class="configure-instance">
+     <div class="configure-instance setup-wizard-heading">
        <form action="${rootURL}/setupWizard/configureInstance" method="post">
           <h1>${%Instance Configuration}</h1>
           <div style="margin: 2em;">
             <table>
             <tr>
               <td><label for="root-url">${%Jenkins URL}:</label></td>
               <td>
-                <input type="text" name="rootUrl" id="root-url" class="root-url" value="" />
+                <input type="text" name="rootUrl" id="root-url" class="jenkins-input" value="" />
```

Comment:
could this be made consistent with the other pages as well please? (so that the label is on top of the form inputs)

<img width="813" alt="image" src="https://user-images.githubusercontent.com/21194782/189595663-ff21647f-60fa-4c2a-a8c3-f43235417c94.png">

the table layout would need replacing with divs most likely